### DISCUSSION TITLE
Account Verification Captcha

### DISCUSSION THREAD

**Discussion by jacobNNHS** [community/community#138308] [New to GitHub]
I have students who have been trying to create an account on Github, but the puzzle captcha will not let them verify their account. They have tried both the dice puzzles and the sound puzzles and keep failing, even if all of the answers are correct. They will get to 19/20 solved puzzles, and the captcha will reset. 

Is there a way to get around this captcha to set up an account? Students need this account for class.

**Comment by akauppi** [comment]
https://github.com/orgs/community/discussions/57317#discussioncomment-14037450

**Comment by alishanawer** [comment]
Unfortunately there isn’t a way to **skip or bypass** the account-creation CAPTCHA, it’s required as part of GitHub’s anti-bot and anti-fraud checks.

However, when users consistently get stuck (e.g. it keeps resetting at 19/20), that usually points to a problem with the **browser or network environment**, not their answers.

---

### What usually fixes this:

* Have students switch to a **different browser** (Chrome, Firefox, Edge, whichever they’re not using)
* Turn **off VPNs, school proxies, or privacy extensions**

  * Many school networks rotate IP addresses mid-session or block GitHub’s validation scripts, which breaks the puzzle
* Try using a **different internet connection** (home Wi-Fi or phone hotspot instead of the school network)
* **Clear browser cookies/cache** before starting the CAPTCHA again

---

### If you are doing this in a classroom setting

GitHub’s CAPTCHA system commonly treats multiple signups from the **same IP range or classroom network** as suspicious, which can cause the puzzles to fail even when solved correctly.

> The simplest workaround for classes is to ask students to create their GitHub account at home or from their **own device and connection**, *before* class.

Hope this helps!
